Source/Blazorise/Components/Dropdown/DropdownItem.razor.cs

```diff
@@ -17,10 +18,23 @@ public partial class DropdownItem : BaseComponent
 
     private bool disabled;
 
+    private bool check;
```

Comment:
Renamed to `@checked`?

Reply:
fair enough, I thought we wouldn't like the necessary @ in `@checked`

Reply:
We already have it in several places. So why not.